sln/WebApiOData.Performance.Official.sln

```diff
@@ -19,12 +19,12 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.CodeAnalysis|Any CPU.ActiveCfg = Release|Any CPU
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.CodeAnalysis|Any CPU.Build.0 = Release|Any CPU
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.Debug|Any CPU.ActiveCfg = Release|Any CPU
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.Debug|Any CPU.Build.0 = Release|Any CPU
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.Release|Any CPU.Build.0 = Release|Any CPU
+		{F7B5A5D5-0B2F-493B-94B2-039B881FD4D3}.CodeAnalysis|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Are these changes required to sln files? Shouldn't the mode remain as Release?

Reply:
That's from VS automatically, I revert it.